src/NHSD.BuyingCatalogue.Ordering.OrderingDatabase/Dockerfile

```diff
@@ -1,18 +1,15 @@
-FROM microsoft/mssql-server-linux:latest
+FROM mcr.microsoft.com/mssql/server:2019-latest
 
-RUN mkdir -p /usr/config
-WORKDIR /usr/config
+WORKDIR ~/config
 
 COPY /ShellScripts .
 COPY /Deployment sql/Deployment
 COPY /Indexes sql/Indexes
 COPY /Tables sql/Tables
 COPY /Security sql/Security
 
-RUN chmod +x /usr/config/entrypoint.sh && chmod +x /usr/config/configure-db.sh
-
-ENTRYPOINT ["./entrypoint.sh"]
+ENTRYPOINT ["/bin/bash", "entrypoint.sh"]
 
 CMD ["tail -f /dev/null"]
 
-HEALTHCHECK --interval=15s CMD /opt/mssql-tools/bin/sqlcmd -U sa -P $SA_PASSWORD -Q "select 1" && grep -q "MSSQL CONFIG COMPLETE" ./config.log
+HEALTHCHECK --interval=15s CMD /opt/mssql-tools/bin/sqlcmd -U sa -P $SA_PASSWORD -Q "select 1"
```

Comment:
Is it not worth checking the script has finished as part of the health check?

Reply:
👍 Thanks for raising this.

When I commited this change, I thought it isn't. However, upon closer inspection, the Healthcheck query will succeed even if the config was not run. 

I suppose since we're used to the db returning healthy when it has been setup (not when its just ready to accept connections) I will return it to how it used to be :)